### ISSUE TITLE
Latest Update Causes Issues on Docker For Mac

### ISSUE THREAD

**Issue by eligundry** [qoomon/docker-host#42]
Hello, we are using Docker Host on our Macs with Docker Compose. One of the developers on my team did a `docker-compose pull` this afternoon and then `docker-compose up` and docker-host started exiting immediately with the following error:

```
[ERROR] docker-host container needs Linux capabilities NET_ADMIN and NET_RAW
  e.g 'docker run --cap-add=NET_ADMIN --cap-add=NET_RAW ...'
```

I then did a `docker-compose pull` and `docker-compose up` on my machine and got the same error. I then looked at the previous Docker tags and pinned my version in Docker Compose to 2.5.4. `docker-compose up` and no error.

Below is a before an after of the service for docker-host in our Docker Composes.

```yaml
  # This exits immediately 
  docker-host:
    container_name: products-docker-host
    image: qoomon/docker-host
    cap_add:
      - NET_ADMIN
      - NET_RAW
```

```yaml
  # This works
  docker-host:
    container_name: products-docker-host
    image: qoomon/docker-host:2.5.4
    cap_add:
      - NET_ADMIN
      - NET_RAW
```

What is weird is that your newest release appears to be just a [documentation change](https://github.com/qoomon/docker-host/commit/ea7df4a08710cc4349058c222a5eb6f6de2e6121) (nifty little star graph!) so maybe something changed in the base image?

**Comment by c2h5oh** [comment]
It's not just mac and it's not just docker-compose.

On Arch Linux with regular docker:
```
# docker run --rm -i -t --cap-add=NET_RAW --cap-add=NET_ADMIN qoomon/docker-host
[ERROR] docker-host container needs Linux capabilities NET_ADMIN and NET_RAW
```

and it's not because those capabilities are not set, because they are:
```
# docker run --rm -i -t --cap-add=NET_RAW --cap-add=NET_ADMIN --entrypoint /bin/sh qoomon/docker-host -c '/usr/sbin/getpcaps $$' 
1: cap_chown,cap_dac_override,cap_fowner,cap_fsetid,cap_kill,cap_setgid,cap_setuid,cap_setpcap,cap_net_bind_service,cap_net_admin,cap_net_raw,cap_sys_chroot,cap_mknod,cap_audit_write,cap_setfcap=eip
```

**Comment by CorySanin** [comment]
Hmmm all my containers that rely on docker-host suddenly started having issues after this update 👀

**Comment by c2h5oh** [comment]
Dockerfile doesn't pin base image - it's simply alpine. Previous build was with alpine 3.12. New one is with 3.13.

**Comment by qoomon** [comment]
@c2h5oh you are right. every commit on master will build a new image with github action and release it a´with the `latest` tag.

Quick workaround pin your version `docker-host:2.5.4`

In the mean while I'll pin the base image as well

**Comment by qoomon** [comment]
`latest` is based on `alpine:3.12` again

**Comment by qoomon** [comment]
Anyone any ideas what change in alpine  version `3.13` causing the problems?

**Comment by qoomon** [comment]
I found the root cause it's because of a new version of `getcaps`. The output format changed. I will fix it today.

**Comment by qoomon** [comment]
Should be fixed for `latest`, `2` and `2.5.5`
